pkg/config/config_template.yaml

```diff
@@ -3830,10 +3830,8 @@ api_key:
 
     ## @param users - list of custom objects - optional
     ## List of SNMPv3 users that can be used to listen for traps.
-    ## NOTE: Currently the Datadog Agent only supports having a
-    ## single user in this list.
     ## Each user can contain:
-    ##  * username     - string - The username used by devices when sending Traps to the Agent.
+    ##  * user         - string - The username used by devices when sending Traps to the Agent.
```

Comment:
If this is an actual change in the configuration format (not just fixing a typo), having a release note to explain it would be nice, and you should probably keep supporting the previous format to avoid breaking changes

Reply:
Yeah that's just fixing a typo in the documentation that was missed 😬

Reply:
Where are the release notes for this change? It broke our production monitoring environment.

Reply:
Hey @ibenrodriguez 
Apologies for the issue you faced, this is unfortunately due to a couple of factors where 1. That `config_template.yaml` was incorrectly referencing `username` instead of `user` and 2. The former version of gosnmp library (that we depend on) was ignoring that username parameter anyway.

`gosnmp` was updated in the Agent 7.51 which now actually checks the username which is the issue you're referring too.
To limit the number of issues we've updated the documentation to use the correct field (the current PR), and we also added transparent backwards compatibility for `username`. These changes will be included in 7.53